components/modal/Modal.component.js

```diff
@@ -30,3 +31,9 @@ const Modalcomp = (props) => {
 };
 
 export default Modalcomp;
+
+Modalcomp.propTypes = {
+  onClose: PropTypes.func,
+  rdsusername: PropTypes.string,
```

Comment:
Will it be good if we write `rdsUsername` instead of `rdsusername` ?

Reply:
Yes, it is definitely more suited. Will change it.